### PR TITLE
Adds API documentation.

### PR DESCRIPTION
**Pull request by grantreighard**

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+> Register object (send): {
+> "username": string,
+> "firstName": string,
+> "lastName": string,
+> "password": string.
+> "role": string,
+> "organizationName": string
+> }
+
+- POST to [https://classroom-angel-development.herokuapp.com/api/login/](https://classroom-angel-development.herokuapp.com/api/login/) - log in a user
```
Comment (johnoro) [inline documentation.md]: I believe this should be `api/users/login`

File: documentation.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+> "id": integer,
+> "username": string,
+> "first_name": string,
+> "last_name": string,
+> "password": string,
+> "role": string,
+> "organization_id": integer
+> }
+
+- POST to [https://classroom-angel-development.herokuapp.com/api/register/](https://classroom-angel-development.herokuapp.com/api/register/) - register a user
```
Comment (johnoro) [inline documentation.md]: I believe this should be `api/users/register`

File: documentation.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+  > "last_in": integer,
+  > "last_out": integer,
+  > "total_minutes_missed": integer,
+  > "expected_hours": integer
+  > }
+
+- POST to [https://classroom-angel-development.herokuapp.com/api/attendance/](https://classroom-angel-development.herokuapp.com/api/attendance/) - create attendance record
+  URL Params | Body | Returns
+  --- | --- | ---
+  N/A | Attendance object (send) | Attendance object (return)
```
Comment (johnoro) [inline documentation.md]: This table looks like it isn't fully formatted!

File: documentation.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
+# Classroom Angel Internal API Documentation
+
+## Teacher Attendance
+
+- GET to [https://classroom-angel-development.herokuapp.com/api/attendance/](https://classroom-angel-development.herokuapp.com/api/attendance/) - retrieve all attendance records
+  URL Params | Body | Returns
+  --- | --- | ---
+  N/A | N/A | All attendance objects (return)
```
Comment (johnoro) [inline documentation.md]: This table looks like it isn't fully formatted!

Comment (grantreighard) [reply documentation.md]: Done

File: documentation.md (hunk 3 above)
Comment (grantreighard) [reply documentation.md]: Done

File: documentation.md (hunk 2 above)
Comment (grantreighard) [reply documentation.md]: Done

File: documentation.md (hunk 1 above)
Comment (grantreighard) [reply documentation.md]: Done

**Comment by grantreighard** [comment]
[Add API docs](https://trello.com/c/gwOjvOIs/61-add-api-docs)

**Pull request merged**